corehq/form_processor/backends/sql/processor.py

```diff
@@ -59,7 +59,8 @@ def is_duplicate(cls, xform, lock):
     @classmethod
     def bulk_save(cls, instance, xforms, cases=None):
         try:
-            XFormInstanceSQL.objects.bulk_create(xforms)
+            for xform in xforms:
+                xform.save()
```

Comment:
do you think the whole function should be wrapped in an atomic section in case attachment saving fails?